packages/lib/src/vis/shared/Tooltip.tsx

```diff
@@ -0,0 +1,37 @@
+import { TooltipWithBounds } from '@visx/tooltip';
+import { type PropsWithChildren } from 'react';
+
+import { type ClassStyleAttrs } from '../models';
+import Overlay from './Overlay';
+import styles from './Tooltip.module.css';
+
+interface Props extends ClassStyleAttrs {
+  show?: boolean;
+  left: number | undefined;
+  top: number | undefined;
+}
+
+function Tooltip(props: PropsWithChildren<Props>) {
+  const { className = '', style, show = true, left, top, children } = props;
+
+  return (
+    <Overlay className={styles.overlay}>
+      {show && children !== undefined && children !== false && (
```

Comment:
Just as before, the tooltip won't show at all if it doesn't have children. Since the `children` can be any `ReactNode` now (including `0`), I check for `undefined` and `false` explicitly.